announce localstack's presence at re:invent 2023 (#152)
localstack/localstack.github.io: content/blog/2023-11-09-join-localstack-at-aws-reinvent-2023/index.md

---
title: Join LocalStack at AWS re:Invent 2023

> description: LocalStack is at AWS re:Invent 2023 — Join us in-person to witness our cutting-edge demos on local Cloud Emulation, Extensions, Chaos Engineering, Security Testing, Ephemeral Environments, CI analytics, and more!
> lead: LocalStack is at AWS re:Invent 2023 — Join us in-person to witness our cutting-edge demos on local Cloud Emulation, Extensions, Chaos Engineering, Security Testing, Ephemeral Environments, CI analytics, and more!

Reviewer: ```suggestion
description: LocalStack is at AWS re:Invent 2023 — Join us in-person to witness our cutting-edge demos on local Cloud Emulation, Extensions, Chaos Engineering, Security Testing, Ephemeral Environments, CI Analytics, and more!
lead: LocalStack is at AWS re:Invent 2023 — Join us in-person to witness our cutting-edge demos on local Cloud Emulation, Extensions, Chaos Engineering, Security Testing, Ephemeral Environments, CI Analytics, and more!
```

date: 2023-11-09
lastmod: 2023-11-09
images: []
contributors: ['LocalStack Team']
tags: ['news']
show_cta_1: true
---

> LocalStack is excited to be participating in AWS re:Invent 2023, Amazon's premier annual cloud computing conference. Amidst keynote addresses by Amazon executives, an array of technical sessions, and a bustling expo floor designed for learning and networking, we're delighted to announce our role as an Exhibit Sponsor for this year's event.

Reviewer: ```suggestion
LocalStack is excited to be participating in [AWS re:Invent 2023](https://reinvent.awsevents.com), Amazon's premier annual cloud computing conference. Amidst keynote addresses by Amazon executives, an array of technical sessions, and a bustling expo floor designed for learning and networking, we're delighted to announce our role as an Exhibit Sponsor for this year's event.
```

Join us at Booth #1074 in the Venetian Expo, Las Vegas, from November 27 to December 1. It's an opportune moment to engage with the LocalStack team, where we'll not only share our insights but also learn valuable lessons from your experiences. 

As the evolution towards empowering developers with local cloud development gains momentum, re:Invent serves as the ideal platform for us to collaborate, exchange ideas, and refine LocalStack based on your feedback. Your journey with LocalStack awaits at AWS re:Invent—let's make it an enriching experience together.

## What can you expect this year?

> At re:Invent this year,you will witness firsthand how we're elevating developer productivity by expediting onboarding and minimizing inefficient dev&test loops that often plague software development and delivery workflows. We will also demonstrate cutting-edge tools and technologies designed to empower teams in testing, debugging, and profiling applications—all without incurring unnecessary cloud costs.

Reviewer: ```suggestion
At re:Invent this year, you will witness firsthand how we're elevating developer productivity by expediting onboarding and minimizing inefficient dev&test loops that often plague cloud software development and delivery workflows. We will also demonstrate cutting-edge tools and technologies designed to empower teams in testing, debugging, and profiling applications—all without incurring unnecessary cloud costs.
```

Talk to us about:

- Streamlining efficiency and reduce waiting times for provisioning cloud development resources by using a fully-local cloud sandbox.
- Conducting sophisticated chaos engineering testing, such as region failover, DNS failover, and service failure simulations, on your local machine.
- Integrating third-party emulators to extend the functionality of LocalStack by adding new services and features via our Extension mechanism.
- Shifting left on IAM Testing to prevent any compromises on security and incurring unnecessary costs due to misconfigurations or inefficiencies.

> - Spinning up LocalStack in ephemeral environments to setup application previews and test your changes before merging them to production.

Reviewer: ```suggestion
- Spinning up LocalStack in ephemeral environments to set up application previews and test your changes before pushing them to production.
```

- Getting insights into your CI builds for cloud infrastructure to improve cloud developer experience across the entire software development lifecycle (SDLC).

> Additionally, LocalStack is now an AWS Partner and has joined AWS Marketplace to grant developer teams the freedom to leverage their AWS budget effectively while harnessing the capabilities of LocalStack’s extensive set of features.

Reviewer: ```suggestion
Additionally, LocalStack is now an AWS Partner and has joined the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-lllzw3ywntoxg) to grant developer teams the freedom to leverage their AWS budget effectively while harnessing the capabilities of LocalStack’s extensive set of features.
```

## Discuss with our Experts!

> At LocalStack, we hold customer engagement in high regard—it's the cornerstone of our mission to empower teams in simplifying the testing and development of cloud applications on their local machines.

Reviewer: ```suggestion
At LocalStack, we hold customer engagement in high regard — it's the cornerstone of our mission to empower teams in simplifying the testing and development of cloud applications on their local machines and in CI pipelines.
```

> You can engage with Executives, Sales, and DevRel to get a one-on-one time with a personalized demo just for you. Fill out the form below and we will follow-up to set a date and time that works for you.

Reviewer: ```suggestion
You can engage with Executives, Sales, and DevRel to get a one-on-one time with a personalized demo just for you. Fill out the form below and we will follow up to set a date and time that works best for you.
```

> ## See you in Las Vegas

Reviewer: ```suggestion
## See you in Las Vegas!
```
